src/app/shared/components/dynamic-form/dynamic-form.component.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { DynamicFormComponent } from './dynamic-form.component';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { PERSONAL_INFO_FORM_CONFIG } from '@features/new-hire/personal-info/personal-info-config';
+
+describe('DynamicFormComponent', () => {
+  let component: DynamicFormComponent;
+  let fixture: ComponentFixture<DynamicFormComponent>;
+
+  const formBuilder: FormBuilder = new FormBuilder();
```

Comment:
Use the same const naming convention everywhere

Reply:
don't understand. can discuss?

Reply:
Agreed